README.md

```diff
@@ -11,9 +11,9 @@ Please follow these steps after `composer install`:
 1) `./flow doctrine:migrate`
 2) `./flow user:create --roles="Neos.Neos:Administrator" <username> <password> <first name> <last name>`
 3) `./flow cr:setup`
-4) `./flow site:create neosdemo Neos.Demo Neos.Demo:Document.Homepage`
-5) `./flow cr:prune`
-6) `./flow cr:import resource://Neos.Demo/Private/Content`
+4) `./flow cr:import resource://Neos.Demo/Private/Content`
+Connect to a Neos site, todo the nodeTypeName parameter is obsolete but necessary
```

Comment:
This is hard to read and understand for anyone just reading this file as it's not a proper sentence. I assume your "todo the nodeTypeName parameter is obsolete but necessary" is just a note for us right now?

Could you please format the readme so that we can distinguish between explantationary comments and our todos.